components/brave_rewards/resources/tip/components/app.tsx

```diff
@@ -30,21 +29,21 @@ export class App extends React.Component<Props, {}> {
     return this.props.actions
   }
 
-  getTipBanner = (url: string, publisher: RewardsTip.Publisher, tweetMetaData?: RewardsTip.TweetMetaData, redditMetaData?: RewardsTip.RedditMetaData) => {
-    if (tweetMetaData) {
+  getTipBanner = (url: string, publisher: RewardsTip.Publisher, mediaMetaData: RewardsTip.MediaMetaData | undefined) => {
+    if (mediaMetaData && mediaMetaData.mediaType === 'twitter') {
```

Comment:
can we add check for undefined `mediaMetaData` and just return? this way we don't need this check

Reply:
done